<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayGame</class>
 <widget class="QWidget" name="PlayGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_end">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_restart">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>65</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>65</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>152</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="First">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_white">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>白子</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_black">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>黑子</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>初级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高级</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
